cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/AbstractDataprocProvisioner.java

```diff
@@ -121,12 +121,13 @@ public final ClusterStatus deleteClusterWithStatus(ProvisionerContext context, C
         }
 
         if (jobDetail != null
-            && jobDetail.getStatus() == RuntimeJobStatus.FAILED
+            && jobDetail.getStatus() != RuntimeJobStatus.COMPLETED
             && (jobDetail instanceof DataprocRuntimeJobDetail)) {
           // Status details is specific to dataproc jobs, so it was not added to RuntimeJobDetail spi.
           String statusDetails = ((DataprocRuntimeJobDetail) jobDetail).getJobStatusDetails();
           if (statusDetails != null) {
-            LOG.error("Dataproc job failed with the status details: {}", statusDetails);
+            LOG.error("Dataproc job {} with the status details: {}",
```

Comment:
nit: `Dataproc job  is in {} state `

Reply:
I think we need to highlight the fact the Term is a enum ( i.e. easy for us to identify and relate to it as a job state/status and not just log line. )